data/schema.ts

```diff
@@ -485,3 +485,15 @@ export const employeesHistory = mysqlTable("employees_history", {
     .notNull()
     .default(""),
 });
+
+export const employeeInboxMessages = mysqlTable("employee_inbox_messages", {
```

Comment:
I expect this to evolve drastically over time:
- note no `employeeUuid` column here. we likely should - I think in Slack's case this translates to every employee being its own "Slack App" which is kind of wild to think about. To short cut this, we're just gonna have the SamePage app receive everything for now. The abstraction to think about here that's useful is imagining SamePage itself being an agent that is sort of a messages representative for all actual employees.
- no `text` column - all raw text we're storing in S3. In the future, they should all ideally be in our AtJson representation if that proves to be useful.
- using `context` as a catchall to save all event data that could be useful